docs/index.rst

```diff
@@ -29,6 +29,10 @@ massive amounts of machine data in real-time.
     the bottom of this page. We regularly update CrateDB to accomodate new
     PostgreSQL clients.
 
+.. important::
+
+    We are also maintaining a more up-to-date list of `CrateDB integration tutorials`_.
```

Comment:
It looks like this does not render well.

-- https://crate-clients-tools--32.org.readthedocs.build/en/32/

Reply:
Changed it to `.. seealso::`.